### PR TITLE
yplanapp supports wildcard TLS now

### PR DESCRIPTION
**Pull request by graingert**

### CODE REVIEW

File: src/chrome/content/rules/yplanapp.com.xml [hunk 1]
```diff
@@ -1,11 +1,12 @@
 <ruleset name="YPlan">
 
-	<target host="yplanapp.com" />
-	<target host="www.yplanapp.com" />
+	<target host="*.yplanapp.com" />
 
-  <securecookie host="^(www\.)?yplanapp.com$" name=".+" />
+	<securecookie host="^.*\.yplanapp\.com$" name=".+" />
```
Comment (graingert) [inline src/chrome/content/rules/yplanapp.com.xml]: does this work with `https://yplanapp.com` as well?

Comment (rugk) [inline src/chrome/content/rules/yplanapp.com.xml]: No. ([just test it by yourself](https://regex101.com/r/jG4bG2/2))
The issue is just that dot `\.` is always required.
But you can e.g. [fix it like this](https://regex101.com/r/jG4bG2/3).

Comment (graingert) [inline src/chrome/content/rules/yplanapp.com.xml]: I got the example from` 	<securecookie host="^.*\.123-reg\.co\.uk$" name=".*" />` is that wrong too?

**Comment by rugk** [comment]
The test currently fails, because you have to add test URLs for wildcard rulesets.
More information: https://github.com/EFForg/https-everywhere/blob/master/ruleset-testing.md

File: src/chrome/content/rules/yplanapp.com.xml (hunk 1 above)
Comment (rugk) [inline src/chrome/content/rules/yplanapp.com.xml]: It [does not](https://regex101.com/r/zG4kM9/1) match the domain without any subdomain (here: `https://123-reg.co.uk`). So if this is the aim of the ruleset it's indeed wrong.

**Comment by graingert** [comment]
@rugk the tests pass now

**Comment by rugk** [comment]
Nice, so you only have to wait until this is merged.

**Pull request merged**